arangodb-net-standard/CollectionApi/GetCollectionCountResponse.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Net;
+
+namespace ArangoDBNetStandard.CollectionApi
+{
+    public class GetCollectionCountResponse
+    {
+        public bool Error { get; set; }
+        public HttpStatusCode Code { get; set; }
```

Comment:
Maybe add a line return in between the properties? That's what we've been doing in all the other models so I think it's nice to keep it consistent. It also helps if in future you need to add one or two comments, you don't end up with an inconsistent formatting.